hack/Makefile

```diff
@@ -0,0 +1,11 @@
+tests = $(shell ./run_test --list 2>/dev/null)
+
+.PHONY: all
+
+all: $(tests)
+
+%:
+	@if [[ "$(tests)" =~ "$@" ]]; then \
+		echo Running test: $@; \
+		VERSION=$(VERSION) IMAGE_NAME=$(IMAGE_NAME) ./run_test $@; \
+	fi;
```

Comment:
What is the check '=~' for?  It guards against misspelled test names?  Also, it is always best to write in POSIX shell if you do Makefiles.